changelog/v1.5.0-beta21/custom-label-comments.yaml

```
@@ -0,0 +1,4 @@
+changelog:
+- type: HELM
```

feels more like a `NON_USER_FACING` change to me
```suggestion
- type: NON_USER_FACING
```